core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -141,19 +141,21 @@ export class AudioBarComponent {
     this.autogeneratedAudioPlayerService.cancel();
   }
 
-  getCurrentAudioLanguageCode(): string {
+  getCurrentAudioLanguageCode(): string | null {
     return this.audioTranslationLanguageService
       .getCurrentAudioLanguageCode();
   }
 
-  getCurrentAudioLanguageDescription(): string {
+  getCurrentAudioLanguageDescription(): string | null {
     return this.audioTranslationLanguageService
       .getCurrentAudioLanguageDescription();
   }
 
   getVoiceoverInCurrentLanguage(): Voiceover {
+    let currentAudioLanguageCode =
+      this.audioTranslationLanguageService.getCurrentAudioLanguageCode();
     return this.audioTranslationManagerService.getCurrentAudioTranslations()[
-      this.audioTranslationLanguageService.getCurrentAudioLanguageCode()];
+      currentAudioLanguageCode === null ? 'null' : currentAudioLanguageCode];
```

Comment:
Why are we setting the string value of null when the currentAudioLanguage code is null?

Reply:
Hi! I get an error saying "Type 'null' cannot be used as an index type." when I don't set the null string to 'null'. Do you have any suggestions about how to fix this without having this line? That would be very helpful. Thank you!

Reply:
Ah I see, in that case, let's add a check such that when `this.audioTranslationLanguageService.getCurrentAudioLanguageCode()` is undefined, we return `null` as the output of `getVoiceoverInCurrentLanguage`.